README.md

```diff
@@ -1,2 +1,43 @@
+| **An Adobe Experience Platform Launch {noun}** | a tags-enabled {noun} | When tags is providing the functionality to the noun, the hyphenated 'enabled' adds clarity to the sentence.  | This extension provides the functionality for adding the 'Media' tracker instance to **an Adobe Experience Platform Launch site** or project.  | This extension provides the functionality for adding the `Media` tracker instance to **a tags-enabled site** or project.   |
+| **The Launch runtime**  |  The tag runtime | Use singular version of Tag and "The" as an article when talking about a general concept/process rather than individual runtime events.  | **The Adobe Experience Platform Launch runtime** (delivered to clients' websites) is designed to support the following browsers:  | ****The tag runtime**** in Adobe Experience Platform (delivered to clients' websites) is compatible with multiple browsers:  |
+| **A Launch property**  | A tag property  | Use the singular version of 'tag' when writing "tag {noun}" in a sentence. | When users install an extension to an Adobe Experience Platform **Launch property**, they will be shown the extension configuration view which your extension will provide.  | When users install an extension to **a tag property**, they will be shown the extension configuration view which your extension will provide.  |
+| **the {noun} extension in Platform Launch**  |  the {noun} tag extension in Platform | The name of the extension will precede the tag qualifier. Later references to tags (Launch) are removed once context has been established.  | Learn about **the Adobe ContextHub extension in Adobe Experience Platform Launch**. | Learn about **the Adobe ContextHub tag extension in Adobe Experience Platform**.  |
+| **Adobe Experience Platform Launch {verb} ...**  | Adobe Experience Platform {verb} | Launch is not always necessary and can be omitted. When the sentence refers to Launch as a service it can often be replaced by referring to Platform. | **Adobe Experience Platform Launch provides** a unified catalog where users can view tag extensions that are available for installation. | **Adobe Experience Platform provides** a unified catalog where users can view tag extensions that are available for installation. |
+| **An edge extension in Launch** | an edge extension OR a tag extension in an edge property  | An edge extension is not to be confused with an edge configuration. Using 'an edge extension' is fine for general use. This example only applies if more context is required.  | Learn how to define an action-type library module for an **edge extension in Adobe Experience Platform Launch.** | Learn how to define an action-type library module for **a tag extension in an edge property**.  |
```

Comment:
Do edge extensions and edge configurations often get confused? I haven't come across this personally... Even so, I don't see what this has to do with branding updates? Edge configurations are no longer a correct term anyway.

Like a lot of others, this is also not a term change. What we _should_ be touching on (in a section on how to refer to extensions), is _tag extensions_ and _event forwarding extensions_.